torch/_inductor/ir.py

```diff
@@ -1690,7 +1690,7 @@ def __init__(self, target: IRNode):
             target.get_device(),
             target.get_dtype(),
             target.get_size(),
-            None,  # type: ignore[arg-type]
+            target.get_stride(),
```

Comment:
Is it make sense to add stride type? now, if we don't add such a parameter, such as a inplace op's post op is convolution, we will call **require_stride1** which will get input's stride and will assert an error.

Reply:
I think there is an issue where the thing we are mutating might be a FlexibleLayout, which means the stride can change after we call this constructor, so copying strides here load lead to subtle bugs..  We could add a `@property` that redirects to the true layout.